Resources/Prototypes/Traits/speech.yml

```diff
@@ -84,6 +82,51 @@
   name: trait-frontal-lisp-name
   description: trait-frontal-lisp-desc
   category: SpeechTraits
-  cost: 2
+  cost: 1
```

Comment:
This is unrelated to your PR, and should be a commented change

Reply:
It was related due to me wanting more variety, but comments have been added (to all of them)